qa/L0_perf_resnet/test.sh

```diff
@@ -137,6 +137,13 @@ done
 # Tests with optimization enabled models
 for MODEL_NAME in $OPTIMIZED_MODEL_NAMES; do
     for PROTOCOL in $PROTOCOLS; do
+        # DLIS-3805: Skip TF-TRT for C API test, it is too inconsistent with
```

Comment:
Does using the server warmup fix this problem?

Reply:
@tanmayv25 do you think it would be OK to add a `model_warmup` to each original model config directly that we copy over from model store, **instead** of doing something like this in the script? It would help with warmup for C API tests, and I don't think it would hurt any existing tests, so no need to add it conditionally.
```
if model == TFTRT:
  echo <model_warmup for "input"> >> config.pbtxt
if model == ORT_TRT:
  echo <model_warmup for "import/input:0"> >> config.pbtxt
if model == libtorch:
  echo <model_warmup for "INPUT__0" and NCHW dims> >> config.pbtxt
```